docs/source/programming/GIN-repositories.md

```diff
@@ -0,0 +1,338 @@
+
+   ```
+   gin login
+   ```
+
+   - You will be prompted for your GIN username and password.
+   - To list the repositories available to your account: `gin repos --all`
+
+:::{tip}
+You may need `sudo` permissions for some of the following `gin` commands. If so, remember to prepend all commands with `sudo`.
+:::
+
+2. **Initialise a GIN repository**
+
+   - **Option 1: on a new directory**
+
+     - Create a new GIN repository locally and on the GIN server by running:
+
+       ```
+       gin create <remote-repository-name>
+       ```
+
+         <details><summary> OR alternatively:</summary>
+
+       Create a repository in the GIN server [from the browser](https://gin.g-node.org/repo/create), and download it locally to your local workspace by running:
+
+       ```
+       gin get <username>/<remote-repository-name>
+       ```
+
+         </details>
+
+     - Once the repository has been initialised, add data to the new local GIN repository with `mv`, `cp` or drag-and-dropping files to the directory.
+
+   - **Option 2: on an existing directory**
+
+     - To create a new repository on the GIN server and in the current working directory in one go, run:
+
+       ```
+       gin create --here <remote-repository-name>
+       ```
+        This will create a repository named `<remote-repository-name>` on the GIN server under your account.
+        <details><summary>OR, to do each step independently:</summary>
+
+       - Initialise the current working directory as a GIN repository by running:
+
+       ```
+       gin init
+       ```
+
+       - Then add a remote for your GIN local repository by running:
+
+       ```
+       gin add-remote <remote-name> <remote-repository-location>
+       ```
+
+       where `<remote-name>` is the name you want to give to the remote (e.g. `origin`) and `<remote-repository-location>` is the location of the data store, which should be in the form of alias:path or server:path (e.g. `gin add-remote origin gin:sfmig/crab-data`).
+
+       - If the remote GIN repository doesn't exist, you will be prompted to either create the remote GIN repository, add the remote address anyways, or abort.
+       - To show the remotes accessible to your GIN account run `gin remotes`.
+       </details>
+
+:::{note}
+Initialising the GIN local repository (with `gin create` or `gin init`) will create a hidden `.git` subdirectory. You can see it on the terminal by running `ls -la` from the local repository. The local repository excluding this `.git` folder is what we call later the _working directory_.
+:::
+
+:::{tip}
+To create a GIN repository on a `ceph` directory:
+
+- You may need to mount the `ceph` directory first. To do this temporarily (i.e., until the next reboot), follow [this guide](https://howto.neuroinformatics.dev/programming/Mount-ceph-ubuntu-temp.html). To do this permanently, follow [this one](https://howto.neuroinformatics.dev/programming/Mount-ceph-ubuntu.html).
```

Comment:
I won't highlight all instance of this, but it's best practice to use explicit targets for internal sphinx links, i.e.: `[this guide](target-mount-ceph-ubuntu-temp)` (see examples of this in the two ceph moutning guides).

This way, the link will still work if we end up moving the files around or if the URL changes.